c7n/resources/cw.py

```diff
@@ -34,7 +34,9 @@ class resource_type(TypeInfo):
         filter_type = 'list'
         name = 'AlarmName'
         date = 'AlarmConfigurationUpdatedTimestamp'
-        cfn_type = config_type = 'AWS::CloudWatch::Alarm'
+        universal_taggable = object()
```

Comment:
Please put back `cfn_type = 'AWS::CloudWatch::Alarm'`

Reply:
please restore the entire line including config_type